manage.py

```diff
@@ -22,6 +22,11 @@
 import sys
 
 if __name__ == "__main__":
+    import sys
+
+    sys.path.insert(0,
+                    '/usr/local/Cellar/gdal/1.11.3_1/lib/python2.7/site-packages')
```

Comment:
@francbartoli That's supposed to be changed into real request. It is for get any feedback about implementation.